website/src/documentation/io/built-in-google-bigquery.md

```diff
@@ -163,6 +163,29 @@ table that you want to write to, unless you specify a [create
 disposition](#create-disposition) of `CREATE_NEVER`. [Creating a table
 schema](#creating-a-table-schema) covers schemas in more detail.
 
+### Data types
```

Comment:
We can preview these changes by looking at staged Website (one of precommit checks). We need to fix formatting here, see: http://apache-beam-website-pull-requests.storage.googleapis.com/8873/documentation/io/built-in/google-bigquery/index.html#data-types.

Reply:
I fixed the problem with the link not displaying correctly, not sure if there is an other issue?